.github/workflows/push-trigger.yml

```
@@ -354,6 +354,10 @@ jobs:
 
   build_dockers_apitest_esignet:
     needs: build_apitest_esignet_local
+    if: >
+      github.event_name == 'push' ||
```

actionlint (1.7.9)

property "detect_changes" is not defined in object type {build_apitest_esignet_local: {outputs: {}; result: string}}

(expression)